components/Forms/Radio.tsx

```diff
@@ -36,14 +36,16 @@ export const Radio: React.VFC<InputProps> = ({
   return (
     <div className="radio">
       <fieldset>
-        <QuestionLabel
-          name={name}
-          type="radio"
-          label={label}
-          requiredText={requiredText}
-          helpText={helpText}
-        />
-        <Tooltip field={name} />
+        <legend>
+          <QuestionLabel
+            name={name}
+            type="radio"
+            label={label}
+            requiredText={requiredText}
+            helpText={helpText}
+          />
+          <Tooltip field={name} />
+        </legend>
```

Comment:
Not sure if legend should wrap the tooltip too

Reply:
resolving convo